config/checkstyle_resources_checks.xml

```diff
@@ -16,10 +16,12 @@
   </module>
   <module name="FileTabCharacter"/>
   <module name="RegexpSingleline">
+    <property name="id" value="noTrailingWhitespace"/>
     <property name="format" value="\s+$"/>
     <property name="message" value="Trailing whitespace"/>
```

Comment:
Please change to "Trailing whitespace is not allowed" for consistency.

Reply:
done